doc/source/mutest.rst

```diff
@@ -119,5 +119,88 @@ used to check that the state of the interface actually changes.
   step("r1", 'vtysh -c "conf t\n interface eth0\n shut"')
   match_step("r1", 'vtysh -c "show interface eth0", "DOWN", "Check for interface DOWN")
 
+An additional json variant of various functions are also available, namely
+:py:func:`match_step_json`, :py:func:`wait_step_json`, and :py:func:`step_json`.
+These functions either return a ``list`` or ``dict`` of the json output of
```

Comment:
Before this addition we don't actually document the return values of the functions. The thinking is to KISS it (i.e., "Keep It Simple 'Silly'") b/c mutest is meant to be usable by folks who might not be actual programmers. I think it would be enough to document here what constitutes a "pass" (i.e., the first of the return tuple is True, but don't say that :) ) and use the API doc comment to document the function return values as you did.